src/DurableTask.SqlServer/Scripts/logic.sql

```diff
@@ -1,6 +1,85 @@
 ﻿-- Copyright (c) Microsoft Corporation.
 -- Licensed under the MIT License.
 
+-- Create custom types. This must be done before creating stored procedures.
```

Comment:
Whenever the app starts up, it checks the version of the schema in the database, which is stored in a `dt.Versions` table. If it doesn't find anything (i.e. a new database), it runs these scripts. The order is `schema-x.y.z.sql` (in logical order), `logic.sql` and finally `permissions.sql`. If it finds a version that's older than the newest version of the `schema-x.y.z.sql` version, then it runs all the higher-versioned `schema-x.y.z.sql` files and then runs `logic.sql`. The latter case is what will happen for existing users that upgrade to the `1.2.0` nuget package.

I guess it would make sense to put this in a README.md file in this directory.

Reply:
Thanks for the explanation!